### PR TITLE
Mention some quirks in tracked-computed interop

### PR DESCRIPTION
**Pull request by boris-petrov**

### CODE REVIEW

**Comment by jenweber** [comment]
Hi, thanks for this! I tagged a couple people to review.

**Comment by chancancode** [comment]
Instead of stating what _doesn't_ work, I think we should just state the positive – if you want to use a getter as a dependent key, you will need to use the `@dependentKeyCompat` decorator. This allows you to refactor existing computed properties into getters without breaking existing code that observes the property.

**Comment by boris-petrov** [comment]
Reworded as suggested by @chancancode. I removed the example because there is pretty much the same one in the `dependentKeyCompat`'s documentation (and I've added a link to it).

File: guides/release/upgrading/current-edition/tracked-properties.md [hunk 1]
```diff
@@ -202,7 +202,12 @@ let person = new Person();
 person.firstName = 'Tom';
 ```
 
-And vice-versa, computed properties used in native getters will autotrack and
+Note, however, that if you want to use a getter as a dependent key, you will
+need to use the [dependentKeyCompat](https://api.emberjs.com/ember/release/functions/@ember%2Fobject%2Fcompat/dependentKeyCompat) decorator. This allows you to refactor
```
Comment (amyrlam) [inline guides/release/upgrading/current-edition/tracked-properties.md]:
```suggestion
need to use the [`dependentKeyCompat`](https://api.emberjs.com/ember/release/functions/@ember%2Fobject%2Fcompat/dependentKeyCompat) decorator. This allows you to refactor
```

this should fix the CI error

**Comment by jenweber** [comment]
Hi! Thanks a bunch for your work and your patience for getting this merged in. It looks like everyone's feedback has been addressed, so let's ship it.

**Pull request merged**